src/main/java/use_case/AccessCalendarData.java

```diff
@@ -44,12 +45,13 @@ public Calendar getCalendar() {
     }
 
     /**
-     *
+     * Adds an event to the calendar and the associated task to the todolist associated with the calendar
      * @param event an event to be added to the calendar
      */
-    public void addEvent(Event event)
+    public void addEvent(Event event, Task task)
```

Comment:
Can access the corresponding task by calling `event.getTask()`. So maybe only need the `event` parameter for this method.

Reply:
Oh right. I'll make the changes.